src/transformers/models/clip/feature_extraction_clip.py

```diff
@@ -124,7 +124,7 @@ def __call__(
         if isinstance(images, (Image.Image, np.ndarray)) or is_torch_tensor(images):
             valid_images = True
         elif isinstance(images, (list, tuple)):
-            if len(images) == 0 or isinstance(images[0], (Image.Image, np.ndarray)) or is_torch_tensor(images[0]):
+            if (len(images) > 0) and (isinstance(images[0], (Image.Image, np.ndarray)) or is_torch_tensor(images[0])):
```

Comment:
Good catch! As @NielsRogge said, this should be fixed for other feature extractors as well.

Reply:
Sure! I'll update once I have more time!